pulpcore/cli/rpm/prune.py

```diff
@@ -0,0 +1,109 @@
+import gettext
+import typing as t
+
+import click
+from pulp_glue.common.context import EntityFieldDefinition, PulpException
+from pulp_glue.rpm.context import PulpRpmPruneContext, PulpRpmRepositoryContext
+
+from pulpcore.cli.common.generic import (
+    PulpCLIContext,
+    pass_pulp_context,
+    pulp_command,
+    resource_option,
+)
+
+_ = gettext.gettext
+
+multi_repository_option = resource_option(
+    "--repository",
+    default_plugin="rpm",
```

Comment:
Give that variable a meaningful name.
```suggestion
    "--repository",
    "repositories",
    default_plugin="rpm",
```